cmd/kubeadm/app/phases/upgrade/postupgrade_test.go

```diff
@@ -149,6 +150,26 @@ func TestWriteKubeletConfigFiles(t *testing.T) {
 	}
 }
 
+func TestRemoveKubeletArgsFromFile(t *testing.T) {
+	tempDir := t.TempDir()
+	testCases := []struct {
+		name          string
+		unwantedFlags []string
+		wantErr       bool
+		wantOut       string
+	}{}
+	for _, tc := range testCases {
+		out := &bytes.Buffer{}
+		err := RemoveKubeletArgsFromFile(tempDir, tc.unwantedFlags, true, out)
```

Comment:
I will add some test cases later. It requires some changes for `dry-run` related codes.